sessions/recap-project-5/art-gallery-app/README.md

```diff
@@ -49,228 +43,291 @@ Download a [Favorite Icon](https://lucide.dev/icons/) and save it in an "assets"
+
+#### Tasks
+
+- [ ] Write tests for the Spotlight component.
+- [ ] Write tests for the Art Piece Preview and List of Art Pieces components.
+- [ ] Write tests for the Navigation component to confirm that links are rendered correctly.
+- [ ] Write tests for the Art Piece Details component to confirm that all details are displayed correctly.
+
+### 7. Comments for Art Pieces
+
+To enhance user engagement, we’ll add a commenting feature that allows users to write and view comments on each art piece. A comment section will be displayed on the details page, showing comments specifically for the art piece being viewed.
 
 #### Value Proposition
 
 **As an** art enthusiast
 
-**I want to** write comments for art pieces
+**I want to** write comments on art pieces
 
-**so that** I can note my ideas regarding the work.
+**so that** I can note down my thoughts and ideas about the work.
 
 #### Acceptance Criteria
 
-- [ ] The detail view has a list of comments for this art piece with the headline "Comments"
-- [ ] Each comment's text is displayed
-- [ ] Each comment's date and time is displayed
-- [ ] The detail view has an input field to write a comment
-- [ ] The detail view has a submit button labeled "Send"
-- [ ] After submitting the form, the comment is appended to the list of comments
+- [ ] The detail view includes a section labeled "Comments" displaying a list of comments for this art piece.
+- [ ] Each comment’s text is displayed.
+- [ ] Each comment’s date and time are displayed.
+- [ ] The detail view includes an input field for writing a comment.
+- [ ] The detail view includes a submit button labeled "Send".
+- [ ] After submitting the form, the new comment is added to the list of comments.
 
 #### Tasks
 
-- [ ] Store comments per art piece in the `artPiecesInfo` state
-- [ ] Create a `CommentForm` component
-- [ ] `CommentForm` props: `onSubmitComment`
-- [ ] Create a `Comments` components
-- [ ] `Comments` props: `comments`
+- [ ] Create at least two new components: one for managing the comment input and submission, and another for displaying the list of comments.
+- [ ] Store the comment information in a state. You can either create a new state specifically for comments or add this information to an existing state, ensuring it’s easily accessible for each relevant art piece.
 
----
+### 8. Persist Favorites and Comments in the Browser
 
-### User Story 8: Persist Favorites and Comments in the Browser
+Currently, all data is saved using `useState`, meaning it’s lost upon page refresh. To improve the user experience, we want to persist data in local storage. Keep in mind that when there is no data in local storage, we’ll need to fetch everything from the API.
 
 #### Value Proposition
 
 **As an** art enthusiast
 
 **I want to** persist my favorites and comments
 
-**so that** the data will not be lost when I close the app.
+**so that** my data remains saved even if I close the app.
```

Comment:
I think we could make this more user-centered as well. For that reason, I suggest the following:

**As an** art enthusiast
**I want** my favorites and comments to be saved,
**so that** my data remains accessible even if I close or refresh the app.